src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -60,13 +60,28 @@ const httpsListen = domain => {
     const config = [];
 
     // HTTPS
-    config.push(['listen', `${domain.server.listenIpv4.computed === '*' ? '' : `${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' : ''}`]);
+    config.push(['listen', 
+        `${domain.server.listenIpv4.computed === '*' ? '' :`${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' :''}`,
+        ]);
```

Comment:
```suggestion
    config.push(['listen', `${domain.server.listenIpv4.computed === '*' ? '' : `${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' : ''}`]);
```
Please revert this change so there is a clean diff